packages/astro/types/content.d.ts

```diff
@@ -89,27 +89,29 @@ declare module 'astro:content' {
 		input: CollectionConfig<S>,
 	): CollectionConfig<S>;
 
-	/** Run `astro sync` to generate high fidelity types */
+	/** Run `astro dev` or `astro sync` to generate high fidelity types */
```

Comment:
I've changed the jsdocs for these because most people will run `astro dev` rather than `astro sync`